### ISSUE TITLE
How to make Air Reactions work on ajax post?

### ISSUE THREAD

**Issue by hoandt** [digitoimistodude/air-reactions#2]
Hi,

I hooked: 
`do_action( 'air_reactions_display', ['post_id' => get_the_ID(),]);`
to my post. Everything works fine but not on posts that loaded via ajax. I don't know how to config the event to make Air Reactions work. 

Could you please  give me any idea?
[https://vnconsumer.com/kham-pha-ninh-thuan-trong-1-ngay-vung-dat-cua-nang-va-gio/] 
(I'm developing a reward system based on Air Reactions, you can see all 'reactions' by click on the "GIFT" button)

**Comment by Niq1982** [comment]
Thanks again for a good issue. Load the newest release (1.0.1) of the plugin and add this to your JS **after** the AJAX has finished loading new content to the page:

```
// After posts loaded from AJAX to DOM
const event = new Event('initAirReactions');
window.dispatchEvent(event);
```

**Comment by hoandt** [comment]
Excellent! Everything works perfectly on 1.0.1! Sorry for bugging around.
